lib/utils/job-utils/smb2-client.js

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+'use strict';
+
+var di = require('di'),
+    XmlDOM = require('xmldom'),
+    SMB2 = require('smb2');
+
+module.exports = Smb2ClientFactory;
+di.annotate(Smb2ClientFactory, new di.Provide('JobUtils.Smb2Client'));
+di.annotate(Smb2ClientFactory, new di.Inject(
+    'Logger',
+    'Promise'
+));
+
+function Smb2ClientFactory(
+    Logger,
+    Promise
+){
+    var logger = Logger.initialize(Smb2ClientFactory);
+    var xmldom,
+        smb2Client,
+        XMLSerializer,
+        serializer;
+
+    function Smb2Client(shareIP, shareName, userName, password){
+        xmldom = XmlDOM.DOMParser;
+        smb2Client = new SMB2({
+            share: '\\\\'+ shareIP + '\\' + shareName,
+            domain: 'DOMAIN',
+            username: userName,
+            password: password
+        });
+        XMLSerializer = XmlDOM.XMLSerializer;
+        serializer = new XMLSerializer();
+    }
+
+    Smb2Client.prototype.readFile = function(fileName){
+        logger.info("Smb2Client readFile function, fileName:" + fileName);
```

Comment:
use `logger.debug` seems to be more reasonable

Reply:
ok